past_projects.md

```diff
@@ -0,0 +1,16 @@
+# Past Projects
```

Comment:
This page will be likely hidden at the beginning as we do not have any projects
Btw should it really "Past projects"? I have the feeling it would be more "Past Gmentees", that may grouped by topics

Reply:
Okay! We can change it to Past Gmentees since we are using project again.